ssl/statem/statem_clnt.c

```diff
@@ -2422,7 +2422,6 @@ MSG_PROCESS_RETURN tls_process_new_session_ticket(SSL *s, PACKET *pkt)
         || (SSL_IS_TLS13(s)
             && (!PACKET_get_net_4(pkt, &age_add)
                 || !PACKET_get_length_prefixed_1(pkt, &nonce)
-                || PACKET_remaining(&nonce) == 0
                 || !PACKET_memdup(&nonce, &s->session->ext.tick_nonce,
```

Comment:
(PACKET_memdup() does not write to the data or length output parameters when PACKET_remaining(pkt) is zero.  This is vaguely tolerable here because the session's extensions block is initialized to zero, but is vaguely disquieting.)